src/configs/price.ts

```diff
@@ -53,12 +52,15 @@ export const DefaultPriceConfig: PriceConfig = {
       weight: 5,
     },
   ],
-  commentElementPricing: {
```

Comment:
I didn't realize that you were able to refer to them with markdown syntax. Are you able to refer to H2 in this syntax?

Reply:
No, if you wanted to refer to H2 in markdown syntax you would have to use JSON:
```
{
  type: 'heading',
  depth: 2,
}
```